content/articles/flutter_speech_recognition_using_alan_voice/index.md

```diff
@@ -0,0 +1,228 @@
+
+
+### Introduction
+Alan voice is an Artificial intelligent powered platform that uses conversational voice to create powerful voice assistance applications. When creating your applications that support speech recognition, changes to the setups are usually made on version releases and updates you make to the initial app build. This is inefficient and hectic to implement. It would be better if the underlying technology that contains the basics for the speech recognition to function well exists and sits locally and whenever any updates need to be made to the functions of our commands, it will be made on a much simpler and easier interface that is easily configurable. With alan voice plugin, we can set up basic functionality in our application that can communicate with users using speech recognition to perform complex function setup on an easy configurable interface provided to us by alan voice. 
+
+To achieve this we will first set up our flutter application, then we will link it to alan voice by integrating alan voice plugin into the application. Next we will test our speech recognition application. Finally we will create/edit/update our commands without updating our application.
+
+
+### Prerequisite
+For this tutorial to take effect, the user must have knowledge of how to create flutter applications or must have created a flutter application before.
+
+### Table of Contents
+- [Introduction](#introduction)
+- [Prerequisite](#prerequisite)
+- [Table of Contents](#table-of-contents)
+- [Goals](#goals)
+- [Setting up our application](#setting-up-our-application)
+- [Creating our Account](#creating-our-account)
+- [Running our application](#running-our-application)
+- [Conclusion](#conclusion)
+- [Further Reference](#further-reference)
+
+### Goals
+We will learn how to setup speech recognition using Alan voice plugin in our flutter application
+See how to set commands and call them in our application inorder to perform some operations.
+
+### Setting up our application
+Head over to your flutter application IDE and spin up a new flutter application and name it any suitable name of your choosing.
+Once your application is set and ready to run we need to integrate flutter alan voice package into our application, this can be obtained from [Flutter package alan_voice](https://pub.dev/packages/alan_voice). Get the latest version of the package and add it to your pubspec.yml file. Ensure the minimumSdkVersion for your application is 21 to check this you can update this in your android/app/build.gradle file.
+
+![set-up-application](/flutter_speech_recognition_using_alan_voice/application.png)
```

Comment:
Please format all your image path like this ! [starting] (/engineering-education/create-a-registration-form-using-react.js-and-connecting-it-with-firebase/start.PNG)